### DISCUSSION TITLE
Shift + Enter creates an entirely new block on mobile

### DISCUSSION THREAD

**Discussion by ghost** [Milkdown/milkdown#149] [General]
Hi there, love what you guys are doing with the editor. I really wanted to thank you for all of your hardwork.  One issue I have is Shift + Enter on mobile goes into a directly new block (the one where you get a double space and a new slash command). This doesn't seem like the case for desktop. So I'm thinking this is a bug?

Also by any chance, are draggable checkboxes possible and is that something you guys are planning to incorporate in the future?

**Comment by Saul-Mirone** [comment]
Hi, I'm wondering how can I input Shift + Enter on mobile?

As for the draggable checkboxes, all blocks are draggable for now on desktop, what you need is use `Cmd/Ctrl + left click` on the block you want to drag, then you can drag it where you want. However, it still have some additional works to do to make it work on mobile.

**Comment by ghost** [comment]
Sorry I meant to say Shift + Return, On iOS it would be the Arrow Up + Return. 

And I didn't know that in regards to the draggable. Can't wait to see it on mobile :) 

Edit: just saw the keyboard shortcut, it'd be considered a `Line Break`. Is there a way to do this on mobile?

**Comment by Saul-Mirone** [comment]
I think maybe that is caused by on mobile keyboard, shift + return is not equal to shift + enter on desktop, it behaves just like a normal return.

**Comment by ghost** [comment]
I see, is there a way to fix and register the shift + return on mobile to behave the same way on desktop?

**Comment by Saul-Mirone** [comment]
I think it depends on the native mobile keyboard, as far as I know, it doesn't support compose key bindings like `Ctrl/Cmd + V`, so I think for mobile, you may need an additional toolbar or some input rules to make it works.
